build/common/gather-logs.sh

```diff
@@ -29,7 +29,7 @@ package_logs(){
     echo $(date) > ${it_results}/runtime.txt
 
     # Look for the FHIR Server Container
-    containerId=$(docker ps -a | grep ibm-fhir-server | cut -d ' ' -f 1)
+    containerId=$(docker ps -a | grep fhir-server | cut -d ' ' -f 1)
```

Comment:
Could there ever be more than one fhir-server active at a time? Just want to check that the name `fhir-server` is now being used for any FHIR server container, and that there's only one active at a time so we don't need to call them different things.

Reply:
not in our CI (at least not that i know of).  i suppose its possible if you're running things locally and you have a different fhir-server image running.  but i don't expect that to be common